.github/workflows/benchmark.yaml

```diff
@@ -2,6 +2,14 @@ name: Benchmark
 
 on:
   pull_request:
+    paths:
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+      - 'rust-toolchain'
+      - 'crates/**'
+      - '!crates/ruff_dev'
+      - '!crates/ruff_shrinking'
```

Comment:
This is where it would be useful to have all internal tools in a single (xtask, dev) directory